<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>View</class>
 <widget class="QMainWindow" name="View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1003</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RobMach</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
<<<<<<< HEAD
     <width>949</width>
     <height>23</height>
=======
     <width>1003</width>
     <height>26</height>
>>>>>>> c4ff94bbf0db489e74a4af36be3499e72d03de24
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionOpenStep"/>
    <addaction name="actionOpenXml"/>
    <addaction name="actionReadPath"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>视图</string>
    </property>
    <widget class="QMenu" name="menuSelect">
     <property name="title">
      <string>Select</string>
     </property>
     <addaction name="actionvertex"/>
     <addaction name="actionedge"/>
     <addaction name="actionmodel"/>
    </widget>
    <addaction name="menuSelect"/>
    <addaction name="actionmodeltree"/>
    <addaction name="actionremove"/>
    <addaction name="actionerase_path"/>
    <addaction name="actionredisplay_path"/>
    <addaction name="actionerase_model"/>
    <addaction name="actionredisplay_model"/>
    <addaction name="actionerase"/>
    <addaction name="actionredisplay"/>
   </widget>
   <widget class="QMenu" name="menuanimation">
    <property name="title">
     <string>动画</string>
    </property>
    <addaction name="actionstart"/>
    <addaction name="actionforward"/>
    <addaction name="actionback"/>
    <addaction name="separator"/>
    <addaction name="actionpause"/>
    <addaction name="actioncontinue"/>
   </widget>
   <widget class="QMenu" name="menucamera">
    <property name="title">
     <string>视角</string>
    </property>
    <addaction name="actionfront"/>
    <addaction name="actionback_2"/>
    <addaction name="actionup"/>
    <addaction name="actiondown"/>
    <addaction name="actionright"/>
    <addaction name="actionleft"/>
   </widget>
   <widget class="QMenu" name="menutool">
    <property name="title">
     <string>工具栏</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuanimation"/>
   <addaction name="menucamera"/>
   <addaction name="menutool"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenXml"/>
   <addaction name="separator"/>
   <addaction name="actionReadPath"/>
   <addaction name="separator"/>
   <addaction name="actionstart"/>
   <addaction name="actionforward"/>
   <addaction name="actionback"/>
   <addaction name="actionpause"/>
   <addaction name="actioncontinue"/>
   <addaction name="separator"/>
   <addaction name="actionfront"/>
   <addaction name="actionup"/>
   <addaction name="actionleft"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionerase_path"/>
   <addaction name="actionredisplay_path"/>
   <addaction name="actionerase_model"/>
   <addaction name="actionredisplay_model"/>
   <addaction name="separator"/>
   <addaction name="actionerase"/>
   <addaction name="actionredisplay"/>
  </widget>
  <action name="actionOpenStep">
   <property name="text">
    <string>打开step</string>
   </property>
  </action>
  <action name="actionvertex">
   <property name="text">
    <string>vertex</string>
   </property>
  </action>
  <action name="actionedge">
   <property name="text">
    <string>edge</string>
   </property>
  </action>
  <action name="actionmodel">
   <property name="text">
    <string>model</string>
   </property>
  </action>
  <action name="actionOpenXml">
   <property name="text">
    <string>打开xml</string>
   </property>
  </action>
  <action name="actionmodeltree">
   <property name="text">
    <string>modeltree</string>
   </property>
  </action>
  <action name="actionReadPath">
   <property name="text">
    <string>读取路径</string>
   </property>
  </action>
  <action name="actionremove">
   <property name="text">
    <string>清除界面</string>
   </property>
  </action>
  <action name="actionstart">
   <property name="text">
    <string>开始</string>
   </property>
  </action>
  <action name="actionforward">
   <property name="text">
    <string>快进</string>
   </property>
  </action>
  <action name="actionback">
   <property name="text">
    <string>快退</string>
   </property>
  </action>
  <action name="actionpause">
   <property name="text">
    <string>暂停</string>
   </property>
  </action>
  <action name="actioncontinue">
   <property name="text">
    <string>继续</string>
   </property>
  </action>
  <action name="actionfront">
   <property name="text">
    <string>正视</string>
   </property>
  </action>
  <action name="actionback_2">
   <property name="text">
    <string>后视</string>
   </property>
  </action>
  <action name="actionup">
   <property name="text">
    <string>俯视</string>
   </property>
  </action>
  <action name="actiondown">
   <property name="text">
    <string>仰视</string>
   </property>
  </action>
  <action name="actionright">
   <property name="text">
    <string>右视</string>
   </property>
  </action>
  <action name="actionleft">
   <property name="text">
    <string>左视</string>
   </property>
  </action>
  <action name="actionerase_path">
   <property name="text">
    <string>隐藏路径</string>
   </property>
  </action>
  <action name="actionredisplay_path">
   <property name="text">
    <string>显示路径</string>
   </property>
  </action>
  <action name="actionerase_model">
   <property name="text">
    <string>隐藏模型</string>
   </property>
  </action>
  <action name="actionredisplay_model">
   <property name="text">
    <string>显示模型</string>
   </property>
  </action>
  <action name="actionerase">
   <property name="text">
    <string>隐藏</string>
   </property>
  </action>
  <action name="actionredisplay">
   <property name="text">
    <string>显示</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
